packages/server-extension/src/index.ts

```diff
@@ -187,6 +193,177 @@ const contentsRoutesPlugin: JupyterLiteServerPlugin<void> = {
   },
 };
 
+/**
+ * A plugin installing the service worker for and handles communication with the Emscpriten file system.
+ */
+const serviceWorkerPlugin: JupyterLiteServerPlugin<void> = {
+  id: '@jupyterlite/server-extension:service-worker',
+  autoStart: true,
+  requires: [],
```

Comment:
Looks like this could be removed if it's empty:

```suggestion
```

Reply:
Also having the service worker logic in its own plugin sounds like a good idea :+1: 

Since it doesn't depend on other services and doesn't provide a token, it could easily be omitted or disabled (which could be useful to https://github.com/executablebooks/thebe-core/pull/3).